widget/widgetAssets/locales/en.json

```diff
@@ -74,7 +74,7 @@
         "SIXTH_PAGE_SUMMARY_HEADING_5": "Automatically install outbreak alert solution packs for outbreak alerts category: ",
         "SIXTH_PAGE_SUMMARY_HEADING_5_1": "severity",
         "SIXTH_PAGE_SUMMARY_HEADING_6": "Following email addresses will receive Outbreak alert update notifications: ",
-        "SIXTH_PAGE_AUTO_INSTALL_HEADING_1": "To install the last 30 days outbreak alert solution pack now, click to ",
+        "SIXTH_PAGE_AUTO_INSTALL_HEADING_1": "To install the last 30 days outbreak response solution pack now, click to ",
```

Comment:
what about other lang files?

Reply:
once all bugs will fixed then will create for other lang files